### PR TITLE
Improvement/update header visual

### PR DESCRIPTION
**Pull request by helloimela**
**Describe pull-request**  
Add header details to the site. 
Mobile first SCSS

**Solving issue**  

Fixes: #112 

**How to test**  
1. Check on mobile and desktop
2. Check requirements in https://github.com/scania/corporate-ui-site/issues/112
3. Compare to Figma >> UI Kit >> Header

**Screenshots**  
Mobile
image
Mobile search open
image
Desktop search open 
image
Desktop
image

**Additional context**  
_Add any other context about the pull-request here._

### CODE REVIEW

**Comment by mmexvr** [comment]
- Header should take 64px plus 1px border in height
- Symbol and search( the box should take 64px total) should be 64px, symbol is now 63px
- Breakpoint should be at md and not sm

File: src/app/components/header/header.component.scss [hunk 1]
```diff
@@ -1,22 +1,130 @@
...
+    border-right: 1px solid var(--sdds-blue-800);
+    transition: all 0.2s ease-in-out;
+    display: flex;
+    align-items: baseline;
+    order: 2;
+
+    input {
+      background-color: transparent;
+      border: none;
+      color: rgba(205, 210, 217, 1); // cannot find colour in scale
```
Comment (helloimela) [inline src/app/components/header/header.component.scss:60]: Colour updated to sdds-blue-800. Need to change the value

Comment (helloimela) [reply src/app/components/header/header.component.scss]: Color updated

**Comment by niklasros** [comment]
The search default text has the wrong colour and text style. Should be set in $detail-02 and $blue-200. 
The position of the search text is off in the Y-axis as well. Might be related due to being set in the wrong text style.

header-search-desktop

**Comment by niklasros** [comment]
The external links are missing a divider that should separate the two links. 
The icons to each external link should be positioned to the right side of the box, and not directly after the link text.

left-nav-external-links

**Comment by niklasros** [comment]
The main links are missing the hover state.
The main link 'Contribute' should have the same height as the rest of the main links.

The state (gray box) for a sub page being active has the wrong width.
The state (gray box) for a sub page should have rounded corners (4px).

left-nav-main-links

left-nav-active

**Comment by niklasros** [comment]
The nav icon is missing the vertical divider on the right side. 
The search icon has the wrong right margin to the browser edge.

header-mobile

**Comment by niklasros** [comment]
I can't access the expanded mobile menu (iPhone 8 w/ Safari).

**Comment by helloimela** [comment]
> I can't access the expanded mobile menu (iPhone 8 w/ Safari).

@niklasros  OK! will take a look at the feedback. Actually the navigation is not working on this pull request. So, you will see the correct style on another pull request later. @mmexvr is working on it 😃

**Comment by mmexvr** [comment]
Yeah, Navigation is on another PR :) @niklasros

**Comment by niklasros** [comment]
When loading in the website, the search icon goes from a big to small size. 

header-loadin-step-1
header-loadin-step-2

**Comment by niklasros** [comment]
The first horizontal divider in the left nav is not aligned with the headers.

header-lines

**Review by mmexvr — APPROVED** [review]

**Pull request merged**